src/Authentication/App/Requests/LoginRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Authentication\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class LoginRequest extends FormRequest
+{
+    public const EMAIL = 'email';
+
+    public const PASSWORD = 'password';
+
+    public function rules(): array
+    {
+        return [
+            self::EMAIL => ['required', 'email:strict'],
```

Comment:
Every time we have a Rule method, we should use it since it provides readability and type-safe

```suggestion
            self::EMAIL => ['required', Rule::email()->strict()],
```